program/src/instruction.rs

```diff
@@ -32,6 +31,8 @@ pub enum LidoInstruction {
         max_validators: u32,
         #[allow(dead_code)] // but it's not
         max_maintainers: u32,
+        #[allow(dead_code)] // but it's not
+        max_validation_fee: u8,
```

Comment:
If we add a new configuration variable, we also need to add an instruction so the manager can change it, and add all the boilerplate in the CLI so we can propose this instruction to the multisig and pretty-print it.

Reply:
It’s fine to do it in a follow-up, that makes this PR easier to review as well. We just shouldn’t forget it.